llvm/test/CodeGen/X86/asm-reg-type-mismatch-avx512.ll

```diff
@@ -5,7 +5,7 @@ define i64 @test1() nounwind {
 ; CHECK-LABEL: test1:
 ; CHECK:       # %bb.0: # %entry
 ; CHECK-NEXT:    #APP
-; CHECK-NEXT:    vmovq {{.*#+}} xmm16 = mem[0],zero
+; CHECK-NEXT:    vmovq 0, %xmm16
```

Comment:
I understand the patch just changes the comments, why codegen affected here too?

Reply:
The instruction comes from inline assembly, when we had the comments in MC we could handle them, but not after moving to MI.

Reply:
Oh, I misunderstood it. The `0` is memory address, I took it as immediate.